rules/Privatization/NodeManipulator/VisibilityManipulator.php

```diff
@@ -58,23 +58,27 @@ public function makeNonAbstract(ClassMethod | Class_ $node): void
     /**
      * @api
      */
-    public function makeFinal(Class_ | ClassMethod | ClassConst $node): void
+    public function makeFinal(Class_ | ClassMethod | Param | ClassConst $node): void
     {
         $this->addVisibilityFlag($node, Visibility::FINAL);
     }
 
     /**
      * @api
      */
-    public function makeNonFinal(Class_ | ClassMethod $node): void
+    public function makeNonFinal(Class_ | ClassMethod | Param $node): void
     {
-        if (! $node->isFinal()) {
+        if (! $this->isFinal($node)) {
             return;
         }
 
         $node->flags -= Modifiers::FINAL;
     }
 
+    public function isFinal(Class_ | ClassMethod | Param $node): bool {
+        return (bool) ($node->flags & Modifiers::FINAL);
+    }
```

Comment:
this is not needed, there is `isVisibility()` method above on line 21., use `isVisibility(Visbility::FINAL)` is enough